src/core/gltf2exporter/gltf2exporter_p.h

```diff
@@ -90,6 +90,19 @@ class KUESA_PRIVATE_EXPORT GLTF2Exporter : public QObject
     Q_PROPERTY(SceneEntity *scene READ scene WRITE setScene NOTIFY sceneChanged)
 
 public:
+    class KUESA_PRIVATE_EXPORT Export
```

Comment:
isn't it implicitely exported already since the parent class is exported

Reply:
indeed !